app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="wrapper">
+  <h1>Find your match!</h1>
+  <h2>Search by gender:</h2>
+  <%= form_tag(search_profiles_path, method: "get") do %>   
+    <%= radio_button_tag(:gender, "male") %>
+    <%= label_tag(:gender_male, "Male", class: 'm-r-2') %>
```

Comment:
It doesn't look like these labels would correspond to the radio buttons here. You could use the `for` attribute on the `label_tag` to ensure that it matches, for better accessibility and easier capybara testing